connector/replicator-with-monitoring/components-destination.yaml

```diff
@@ -76,7 +76,7 @@ spec:
   tls:
     autoGeneratedCerts: true
   image:
-    application: confluentinc/cp-schema-registry:7.0.1
+    application: confluentinc/cp-schema-registry:7.1.0
     init: confluentinc/confluent-init-container:2.2.0-1
```

Comment:
should we switch all the init containers to 2.3?

Reply:
I thinks so. I think I missed `2.2.0-1`s. I will search again.